airflow/www/static/js/dag/details/taskInstance/taskActions/ClearInstance.tsx

```diff
@@ -242,21 +243,23 @@ const ClearInstance = ({
   useKeysPress(keyboardShortcutIdentifier.taskRunClear, onOpen);
 
   const clearLabel = "Clear and retry task.";
+  const canEdit = getMetaValue("can_edit") === "True";
```

Comment:
`canEdit` is specified twice. Let's delete one

Reply:
Ok seems right

Reply:
Coming back to this, after some rethinking canEdit and canEditTaskInstance should just be declared once at the start of the file. This changes were only made for being able to create tests, which I find them not that necessary since these changes are so small. Should I remove the tests files and move the declarations to the top of the file? What do you think?